framework/components/ecl-site-switchers/ecl-site-switchers.config.js

```diff
@@ -35,11 +40,16 @@ module.exports = {
       label: 'Footer',
       context: {
         variant: 'footer',
-        political: { href: '#', label: 'Commission and its priorities' },
+        political: {
+          href: '#',
+          label: 'Commission and its priorities',
+          variant: '',
+        },
         info: {
           href: '#',
           label: 'Policies, information and services',
           is_active: true,
+          variant: '',
```

Comment:
is it necessary to define `variant: ''`?

Reply:
yes, it is.
That's something I mentioned month ago, if i don't define it here, it will somehow "inherit" the variant set before ("header" or "footer" in our case). So it doesn't break anything, but add a `ecl-link--footer` class for instance, which doesn't exists.

Reply:
ok, makes sense

if we had time, we'd refactor the code and use `only` more frequently (on include/embed)